test/run_exodiff_test.py

```diff
@@ -25,9 +102,11 @@ def runtestdiff(executable_name, cli_flag, input_deck_name, num_ranks, use_openm
     #
     command.append(executable_name)
     #
+    command.append(input_deck_name)
     if cli_flag:
         command.append("--" + cli_flag)
-    command.append(input_deck_name)
+        if "use_vt" in cli_flag:
+            command.append("--vt_no_terminate")
```

Comment:
On my machine at least the terminate handler seems to cause some issues so I added this